sdk/src/main/java/io/opentelemetry/sdk/trace/RecordEventsSpanImpl.java

```diff
@@ -191,7 +226,7 @@ long getEndNanoTime() {
    */
   long getLatencyNs() {
     synchronized (this) {
-      return hasBeenEnded ? endNanoTime - startNanoTime : clock.nowNanos() - startNanoTime;
+      return getEndNanoTime() - startNanoTime;
```

Comment:
Should we avoid double sync here? maybe have a private getEndNanoTimeUnlocked? Also for the moment this doesn't seem to be used, maybe add a test for this.

Reply:
Make sense, updated and added tests.